packages/react/src/FilteredActionList/useAnnouncements.tsx

```diff
@@ -91,30 +64,11 @@ export const useAnnouncements = (
 
       liveRegion?.clear() // clear previous announcements
 
-      if (items.length === 0 && !loading) {
-        announce('No matching items.', {delayMs})
-        return
-      }
```

Comment:
we don't need to announce no matching items anymore? or we're getting that implicitly?